backends-common/cassandra/pom.xml

```diff
@@ -130,24 +131,31 @@
                     <groupId>org.apache.james</groupId>
                     <artifactId>james-server-util-java8</artifactId>
                 </dependency>
-		        <dependency>
-		            <groupId>com.datastax.cassandra</groupId>
-		            <artifactId>cassandra-driver-core</artifactId>
-		            <version>2.1.8</version>
-		        </dependency>
-		        <dependency>
-		            <groupId>com.google.guava</groupId>
-		            <artifactId>guava</artifactId>
-		        </dependency>
-			<dependency>
-			    <groupId>commons-lang</groupId>
-			    <artifactId>commons-lang</artifactId>
-			    <version>2.6</version>
-			</dependency>
-            <dependency>
-                <groupId>javax.inject</groupId>
-                <artifactId>javax.inject</artifactId>
-            </dependency>
+                <dependency>
+                    <groupId>com.datastax.cassandra</groupId>
+                    <artifactId>cassandra-driver-core</artifactId>
+                    <version>2.1.8</version>
+                </dependency>
+                <dependency>
+                    <groupId>io.netty</groupId>
+                    <artifactId>netty-transport-native-epoll</artifactId>
+                    <version>4.1.10.Final</version>
+                    <classifier>linux-x86_64</classifier>
```

Comment:
no, that's a bad idea, we should not build for a single platform. If there's not way to make something universal, then we need to add the dependency at deploy time (or in docker image because docker is x86-64 anyway)